contrib/rpm/polybar.spec

```diff
@@ -0,0 +1,70 @@
+#
+# spec file for package polybar
+# Initially created for openSUSE
+#
+
+Name:           polybar
+Version:        3.3.0
+Release:        0
+Summary:        A fast and easy-to-use status bar
+License:        MIT
+Group:          System/GUI/Other
+URL:            https://github.com/jaagr/polybar
+Source:         https://github.com/jaagr/polybar/archive/%{version}.tar.gz
+BuildRequires:  clang >= 3.4
+BuildRequires:  cmake >= 3.1
+BuildRequires:  pkgconfig
+BuildRequires:  python-xml
+BuildRequires:  xcb-util-image-devel
+BuildRequires:  xcb-util-wm-devel
+# optional dependency
+BuildRequires:  pkgconfig(alsa)
+# main dependency
+BuildRequires:  pkgconfig(cairo)
+BuildRequires:  pkgconfig(jsoncpp)
+BuildRequires:  pkgconfig(libcurl)
+BuildRequires:  pkgconfig(libpulse)
+BuildRequires:  pkgconfig(python)
+BuildRequires:  pkgconfig(xcb)
+BuildRequires:  pkgconfig(xcb-proto)
+BuildRequires:  pkgconfig(xcb-util)
+%if 0%{?suse_version} <= 1315
+BuildRequires:  i3-devel
+%else
+BuildRequires:  i3-gaps-devel
+%endif
+%if 0%{?suse_version}
+BuildRequires:  libiw-devel
+%else
+BuildRequires:  wireless-tools-devel
+%endif
+
+%description
+A fast and easy-to-use status bar for tilling WM
+
+%prep
+%setup -q
+
+%build
+%cmake
+make userconfig
```

Comment:
This puts a file into the user's home directory (and overwrites existing ones). I don't know if that belongs into a package build file.

Reply:
Thanks for pointing this out. In our case this actually does nothing then, because the OBS is a clean build environment. Changes to a users home are just ignored in this scenario. It seems sysek was not aware of this, I'll update this PR and also the internal package.